ckanext/datarequests/constants.py

```diff
@@ -35,3 +35,4 @@
 DESCRIPTION_MAX_LENGTH = 1000
 COMMENT_MAX_LENGTH = DESCRIPTION_MAX_LENGTH
 DATAREQUESTS_PER_PAGE = 10
+CLOSE_CIRCUMSTANCE_MAX_LENGTH = 100
```

Comment:
is 100char enough?

Reply:
lets go with 255 just in case and as you said its a common default.

Reply:
Increased to 255